certbot/tests/renewupdater_test.py

```diff
@@ -75,6 +62,68 @@ def test_deployer_skip_dry_run(self, mock_log):
         self.assertEquals(mock_log.call_args[0][0],
                           "Skipping renewal deployer in dry-run mode.")
 
+    @mock.patch('certbot.plugins.selection.choose_configurator_plugins')
+    @test_util.patch_get_utility()
```

Comment:
I don't think we need to call `patch_get_utility` here or on any of the tests below. I just ran tests with them removed and they still passed.